nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -22,37 +22,39 @@
  * 1. Models are limited to creating with configuration files.
  * 2. sub-plugin receives only pre-processed tensor data.
  * 3. The current feature behavior has been tested with MNIST.
+ * 4. mnist.json has 'gst_caps' containing the information below.
+ * "gst_caps":"other/tensors, format=(string)static, framerate=(fraction)0/1,
+ *  num_tensors=(int)2, dimensions=(string)1:1:784:1.1:1:10:1,
+ *  types=(string)float32.float32"
  *
  * ## Example launch line is as below
- * gst-launch-1.0 datareposrc location=mnist_trainingSet.dat \
+ * gst-launch-1.0 datareposrc location=mnist_trainingSet.dat json=mnist.json \
```

Comment:
@songgot 

Isn't the property name ```json``` strange?
```json``` is just a file format for the data repository metadata.

It'd be more meaningful if it's ```metadata=mnist.json```. You are accepting metadata in the format of json.

Reply:
I actually thought about what to write the property name as. And I found somewhere to use json with ```--json=```.
I will consider  it. 
thank you.